build.gradle

```diff
@@ -76,7 +76,7 @@ allprojects {
     }
 }
 
-def os = osdetector.classifier.replace("osx", "macosx").replace("windows-x86_32", "windows-x86")
+def os = osdetector.classifier.replace("osx", "macosx").replace("windows-x86_32", "windows-x86").replace("x86_32", "x86")
```

Comment:
Isn't this equivalent to  `def os = osdetector.classifier.replace("osx", "macosx").replace("x86_32", "x86")`?